MetaMask/metamask-docs
# feat(docs): added rpc-parser component for reference pages

## Description

- Created a custom react component/parser for OpenRPC and used it for JSON-RPC methods pages.
- This PR does **NOT** include a form for configuring parameters and sending a request (after the review, the test page **will be hidden** for the PROD)

## How to test:

Go to the test page for the `eth_call` method - https://docs.metamask.io/feat-reference-pages/wallet/reference/new-reference/ and check the main layout (description, label, request/response sections)

## Links:

[JIRA Issue](https://infura.atlassian.net/browse/ACT-1357)
[Figma](https://www.figma.com/design/gBbfVUyOdjS5OYZcg9mbUX/Sandbox-%2F-Interactive-API?node-id=492-23584&t=KnAPNGAeBy8pGawC-0)
